### ISSUE TITLE
Cancel Roadbuilding with Esc Key

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#6284]
<img align='left' src='https://codeberg.org/avatars/a4ad2f9a2504f052397d8ba82ed48ba1' alt='Nordfriese' height='48' />*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/4640)*
*Created on Sun Dec 31 11:10:26 CET 2023 by **Benedikt Straub (Nordfriese)***

---

When in road/waterway building mode, it should be possible to cancel road building with the <kbd>Esc</kbd> key. This would be consistent with other uses of Esc to cancel/close something.
